tools/gromacs/sim.xml

```diff
@@ -98,16 +98,28 @@
                     pme_order      = 4        ; cubic interpolation
                     fourierspacing  = 0.16    ; grid spacing for FFT
                     ; Temperature coupling is on
+                    #if $sets.mdp.couplinggroups.wholesystem =="true":
+                    tcouple   = V-recale ; modified Berendsen thermostat
+                    tc-grps = System ; coupling the whole system. Can be used in any system, beyond peptides. 
+                    tau_t    = 0.1 ; time constant in ps
+                    ref_t    = $sets.mdp.temperature
+                    #else
                     tcoupl    = V-rescale              ; modified Berendsen thermostat
-                    tc-grps    = Protein Non-Protein  ; two coupling groups - more accurate
+                    tc-grps   = Protein Non-Protein  ; two coupling groups - more accurate
                     tau_t    = 0.1    0.1          ; time constant, in ps
                     ref_t    = $sets.mdp.temperature $sets.mdp.temperature           ; reference temperature, one for each group, in K
+                    #end if
                     ; Periodic boundary conditions
                     pbc    = xyz    ; 3-D PBC
                     ; Dispersion correction
                     DispCorr  = EnerPres  ; account for cut-off vdW scheme
+                    #if $sets.ensemble =="nvt" and $sets.mdp.startingvelocities.startvel =="true":
```

Comment:
I think if you implement it this way, you should make it clear in the user interface that the option only works when the NVT option is selected.

Reply:
It would be a use-case for a conditional, so that the option only appears when NVT is selected.

Reply:
made a conditional in sim.xml instead, where the starting velocities option only comes up if a person chooses NVT.